src/test/java/edu/kis/vh/nursery/FIFORyhmerTest.java

```diff
@@ -7,7 +7,7 @@
 
 public class FIFORyhmerTest {
 	FIFORyhmer ryhmer = new FIFORyhmer();
-	final int IS_EMPTY_INDICATOR = -1;
+	final int IS_EMPTY_INDICATOR = 0;
```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).